src/mention/mention-created.event.ts

```diff
@@ -0,0 +1,13 @@
+import type { Mention } from 'mention/mention.entity';
```

Comment:
I don't think we need the new `mention.entity.ts`. A good way is to follow the pattern that we have for [post-liked-event](https://github.com/TryGhost/ActivityPub/blob/main/src/post/post-liked.event.ts). This has a post and an accountId. We can have similar for mentions. 

`async createMentionNotification(post: Post, accountId: number)`

We also don't need the entity for saving mentions in db so this won't be reused any where else in the code.

Reply:
updated in 731d4d4d24dd984e00f9b9bead0a6c1ac8b463d8